service/server.py

```diff
@@ -63,14 +61,16 @@ def check_field_vals(field_vals):
             'into respective parts. Expected street_PAON_SAON.', 404))
 
 
-def get_query_dict(field_vals):
+def get_query_dict(address_rec):
```

Comment:
Not important, but given we have to resolve conflicts, anyway, perhaps we could rename this function to 'get_PPI_query_param_dict' or sth similar. May be a bit too generic as it is.